apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -62,197 +61,160 @@ var (
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.1-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.3-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.3-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.4-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.4-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.5-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.5-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.6-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.6-nanoserver-1803"),
-			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
+			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.2-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.1-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.1-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.4-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.4-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
-			CanalCNIBinaries:   m("rancher/canal-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.4-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.4-nanoserver-1803"),
-			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
-			CanalCNIBinaries:   m("rancher/canal-cni:v0.0.1-nanoserver-1803"),
+			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.2-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
```

Comment:
Why the image is 1803? it should be 1809?

Reply:
`-1803` is a suffix for now, `rancher/rancher` can modify it by Windows OS version. When supporting `manifest`, I will change this.

Reply:
I think you will need new map for windows images instead of using k8s version as key. Then you can support both 1803 and 1809.

Reply:
And I think we need to move to v1.14 .